features/shop/wishlist/customer/clearing_wishlist.feature

```diff
@@ -0,0 +1,41 @@
+@wishlist_as_customer
+Feature: Clearing a wishlist
+    In order to clear a wishlist
+    As a customer
+    I want to be able to clear wishlists
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "Harness climbing" priced at "$30.00"
+        And the store has a product "Rope" priced at "$40.00"
+        And there is a customer "Alex Honnold" identified by an email "ahonnold@climb.com" and a password "dlonnoh1"
+        And user "ahonnold@climb.com" has a wishlist named "For Alex"
+        And there is a customer "Adam Ondra" identified by an email "aondra@climb.com" and a password "ardno1"
+        And user "aondra@climb.com" has a wishlist named "For me"
+        And I create a JWT Token for customer identified by an email "aondra@climb.com"
+
+    @graphql
+    Scenario: Cleaning a wishlist
+        Given user have a product "ROPE" in my wishlist "For me"
```

Comment:
user has

Reply:
You have the literal issue in a few places. Could you localize them and fix also?